packages/beacon-node/src/metrics/metrics/beacon.ts

```diff
@@ -109,6 +113,11 @@ export function createBeaconMetrics(register: RegistryMetricCreator) {
         name: "beacon_fork_choice_indices_count",
         help: "Current count of indices in fork choice data structures",
       }),
+      notReorgedReason: register.gauge<{reason: NotReorgedReason}>({
+        name: "beacon_fork_choice_not_reorged_reason_total",
```

Comment:
do we have panels to visualize those metrics? can be done in a follow-up PR but I rather prefer having those as part of the PR to avoid having unused metrics

Reply:
> do we have panels to visualize those metrics? can be done in a follow-up PR but I rather prefer having those as part of the PR to avoid having unused metrics

No I don't think so. I was thinking of stuffing this inside the block production panel but I think it's better to have a new dashboard to display this and other reorg metrics. Unfortunately adding new panel will be too much for this PR.

Reply:
prefer having a separate PR for grafana panels, as long as @ensi321 doesn't miss it

Reply:
let's create an issue for it so we don't forget